packages/charts/src/components/MicroBarChart/MicroBarChart.tsx

```diff
@@ -61,6 +61,10 @@ export interface MicroBarChartProps
    * - `DataLabel`: a custom component to be used for the data label
    */
   measure: MeasureConfig;
+  /**
+   * An optional number for the maxValue of the valueBar
+  */
```

Comment:
Please also mention that if this prop is not set, the maximum value is equal to the highest number of the corresponding accessor in the dataset.